<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MuGen</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Generate</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_generate">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Key:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Length:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_key">
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C#</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>D</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>D#</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F#</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G#</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A#</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Tempo :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Export to:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBox_mode">
            <item>
             <property name="text">
              <string>major</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>minor</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_filename">
            <property name="text">
             <string>MuGen.mid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_length">
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;number of chords that will be generated&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>16</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Chords              </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_tempo">
            <property name="text">
             <string>80</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>bpm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Solve</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="8" column="1" colspan="3">
       <widget class="QPushButton" name="pushButton_solve">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox_mode_2">
        <item>
         <property name="text">
          <string>major</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>minor</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Export to:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Key:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBox_key_2">
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_filename_import">
        <property name="text">
         <string>CMajBassLine.mid</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_filename_export">
        <property name="text">
         <string>MuGen.mid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Import bassline from:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_out">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
